server.js

```diff
@@ -214,7 +214,15 @@ class StoreServer {
       name: key
     }
     try {
-      manifest = await pda.readManifest(archive)
+      
+      // https://javascript.info/promise-api#promise-race
+      // give Paul's Dat API a moment to resolve and win the race
+      // otherwsie, just return the resolved manifest you've already got
+      manifest = Promise.race([
+        await pda.readManifest(archive),
+        new Promise((resolve, reject) => setTimeout(() => resolve(manifest), 500))
```

Comment:
This might be a good place to require the `delay` module and have something like `delay(500).then(() => manifest)`

Also, we should probably have the timeout be configurable in the options